forwards_paper.tex

```diff
@@ -215,31 +215,47 @@ \subsection*{Tree sequences}
 in Figure~\ref{fig:example_tree_sequence}, illustrated by red hexagons. The first
 mutation occurs at site zero (which is in the left-hand tree), which is a simple
 mutation resulting in node $2$ inheriting the state 'T'. The second side (in the
-right hand tree) has two mutations: one occuring over node $3$ changing the state to
+right hand tree) has two mutations: one occurring over node $3$ changing the state to
 `C' and a back mutation over node $1$ changing the state to `G'.
 \jk{Not sure if I'm labouring the point here: this should be reasonably obvious.
 Perhaps if we added the state changes to the trees, and the states of the leaves
 below the samples, we could get rid of all the example discussion here?}
 
-
-\jk{Other stuff we can review here:}
-\begin{enumerate}
-\item Encoding a tree sequence topology like this is very efficient, much, much more compact than newick.
-Can refer to example in msprime paper, or could do a large running example here. Say 500K samples of
-human CHR1, something like that. Newick will be hundreds of TB.
-
-\item Can also make the point that it's much more efficient than, say, VCF. We could drop some
-mutations on the example above, and then report how big it would be if encoded with VCF and BCF.
-
-\item The \msprime\ API is very efficient for working with these large datasets. Examples on
-how long it takes to iterate over all the trees, compute pi, etc.
-
-\item The point of all this is to emphasise that not only is it efficient to generate the
-data as a tree sequence, it's also very efficient to store and process it this way too. By the
-time we get to the end of this section we should have convinced the reader of this.
-
-\end{enumerate}
-
+This encoding of a sequence of trees and accompanying mutational information is
+very concise. To illustrate this, we ran a simulation of $500,000$ samples of a
+$200$ megabase human-like chromosome ($N_e=10^4$ and per-base mutation and
+recombination rates of $10^{-8}$ per generation) using \msprime. This resulted
+in about 1 million distinct marginal trees and $1.1$ million infinite-sites
+mutations. The HDF5 file encoding the nodes, edges, sites and mutations (as
+described above) for this simulation consumed 157MiB of storage space. Using
+the \msprime\ Python API, the time required to load this file into memory was
+around 1.5 seconds, and the time required to iterate over all 1 million tree
+was 2.7 seconds. In contrast, recording the topological information in Newick
```

Comment:
Should we see if the journal accepts emojis? :excited_face:  Seriously, this is great to emphasize here.